caravel/assets/javascripts/dashboard.jsx

```diff
@@ -15,6 +16,8 @@ require('./caravel-select2.js');
 require('../node_modules/react-grid-layout/css/styles.css');
 require('../node_modules/react-resizable/css/styles.css');
 
+require('datatables.net-bs');
+require('../node_modules/datatables-bootstrap3-plugin/media/css/datatables-bootstrap3.css');
```

Comment:
what's the difference between these two imports? are they both css modules for datatables?